src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -221,6 +221,18 @@
     <value>Profile that opens when clicking the '+' icon or by typing the new tab key binding.</value>
     <comment>A description for what the default profile is and when it's used.</comment>
   </data>
+  <data name="Globals_DefaultTerminal.Header" xml:space="preserve">
+    <value>Default terminal</value>
+  </data>
+  <data name="Globals_DefaultTerminal.HelpText" xml:space="preserve">
+    <value>The terminal application that launches when a command-line application is run without an existing session, like from the Start Menu or Run dialog.</value>
+  </data>
+  <data name="Globals_DefaultConsole.Header" xml:space="preserve">
+    <value>Default console (PTY)</value>
+  </data>
+  <data name="Globals_DefaultConsole.HelpText" xml:space="preserve">
+    <value>The console engine and pseudoconsole version that will be used to power the default terminal application.</value>
+  </data>

```

Comment:
Be sure to put comments on all of these (like the rest of the file). Localization team said that the reason most of the Settings UI strings weren't localized well is because there weren't comments, and I'll forever be haunted by this experience hahaha